### PR TITLE
add lacaml <= 7.6.2 constraint to oml

### PR DESCRIPTION
**Pull request by struktured**
See https://github.com/hammerlab/oml/issues/151

### CODE REVIEW

**Comment by dsheets** [comment]
Thanks!

It looks like `lacaml` 7.1.5 is too old for `oml` 0.0.1 but 7.2.3 is recent enough (actual bound somewhere between). It also looks like `oml` 0.0.1 may be unbuildable due to ml/mli incompatibility:

```
# File "src/lib/functions.ml", line 1:
# Error: The implementation src/lib/functions.ml
#        does not match the interface src/lib/functions.cmi:
#        Values do not match:
#          val regularized_lower_gamma : a:float -> float -> float
#        is not included in
#          val regularized_lower_gamma : float -> float -> float
#        File "src/lib/functions.ml", line 13, characters 4-27:
#          Actual declaration
# Command exited with code 2.
```

**Comment by rleonid** [comment]
@struktured Sorry for the pain, you have to add the ocephes constraint `{< "0.8"}` to 0.0.2 and 0.0.3 as well. `0.0.5` is the first version for ocephes.0.8

**Comment by struktured** [comment]
@rleonid  @dsheets I see a depext dependency failure for lbfgs. Is that due to an opam misconfiguration or is this expected?

**Comment by rleonid** [comment]
I think this is a misconfiguration of `lbfgs`'s `lacaml` depopt. `lacaml.7.2.4` doesn't build the library necessary to build the (test) programs that are built with `lbfgs`. (See the update to lacaml [7.2.5](https://github.com/mmottl/lacaml/commit/3bb15fc6a5920f224ce44df7dbaa16534ac7a4a0)).

**Comment by rleonid** [comment]
@dsheets I'm having a hard time figuring what happened in the the one remaining instance that failed (4.02, 1.2). But for some reason it installs `lacaml.7.2.4` again to check `lbfgs`.

File: packages/oml/oml.0.0.5/opam [hunk 1]
```diff
@@ -10,8 +10,8 @@ install: [make "install"]
 remove: ["ocamlfind" "remove" "oml"]
 depends: [
   "ocamlfind" {build}
-  "lacaml"
+  "lacaml" {>= "7.2.3" & <= "7.6.2"}
```
Comment (dsheets) [inline packages/oml/oml.0.0.5/opam]: This is the incorrect constraint. `7.2.3` appears to build but `7.2.4` is missing a `.a`.

Comment (rleonid) [inline packages/oml/oml.0.0.5/opam]: Ah yes! That should also say `7.2.5`.

Comment (struktured) [inline packages/oml/oml.0.0.5/opam]: Updated accordingly (btw, I thought "early" oml versions meant < 0.5, not <= 0.5. My bad)

**Comment by rleonid** [comment]
`\o/`

**Pull request merged**